aptos-move/move-examples/parallel_accounts/sources/parallel_accounts_demo.move

```diff
@@ -0,0 +1,129 @@
+/// A demo of how multiple accounts can be used to delegate to a single resource
+/// account for parallelism between transactions.
```

Comment:
Technically though, transfers like in the demo wouldn't be parallelized anyway as the coins are all coming from the same resource account

Reply:
Okay, technically no.  But, keeping in mind that you have multiple fee payers on one account.  If it had resources that were non-conflicting, then it wouldn't be an issue.

Reply:
Maybe call this shared or high throughput and add a todo or note on why it isn't parallel. Also we should share this with the blockchain as this likely breaks their assumption with transaction shuffling

Reply:
this needs a lot more documentation or a readme, but it is a good start.